Mail/Views/Settings/SettingsView.swift

```diff
@@ -89,23 +89,27 @@ struct SettingsView: View {
 
                     // MARK: Sync Calendar/Contacts
 
-                    Button {
-                        matomo.track(eventWithCategory: .syncAutoConfig, name: "openFromSettings")
-                        if platformDetector.isMac {
-                            isShowingSyncProfile = true
-                        } else {
-                            mainViewState.isShowingSyncProfile = true
+                    if !platformDetector.isMac {
+                        Button {
+                            matomo.track(eventWithCategory: .syncAutoConfig, name: "openFromSettings")
+                            if platformDetector.isMac {
+                                isShowingSyncProfile = true
+                            } else {
+                                mainViewState.isShowingSyncProfile = true
+                            }
+                        } label: {
+                            SettingsSubMenuLabel(title: MailResourcesStrings.Localizable.syncCalendarsAndContactsTitle)
                         }
-                    } label: {
-                        SettingsSubMenuLabel(title: MailResourcesStrings.Localizable.syncCalendarsAndContactsTitle)
+                        .buttonStyle(.plain)
                     }
-                    .buttonStyle(.plain)
 
                     // MARK: AI Writer
 
-                    if featureFlagsManageable.isEnabled(.aiMailComposer) {
-                        SettingsSubMenuCell(title: MailResourcesStrings.Localizable.aiPromptTitle, subtitle: aiEngine.title) {
-                            SettingsAIEngineOptionView()
+                    if !platformDetector.isMac {
+                        if featureFlagsManageable.isEnabled(.aiMailComposer) {
```

Comment:
```suggestion
                    if !platformDetector.isMac && featureFlagsManageable.isEnabled(.aiMailComposer) {
```

Reply:
After reviewing the rest of the code I'm wondering if we could use the FeatureFlagManager to do this. 
We'll discuss it with the rest of the team.